test/test_lt.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+import pytest
+from sympy import Symbol, symbols
+from galgebra.ga import Ga
+from galgebra import utils
+
+class TestLt(unittest.TestCase):
+
+    # reproduce gh-105
+    def test_lt_matrix(self):
+        base = Ga('a b', g=[1,1], coords=symbols('x,y',real=True))
+        a,b = base.mv()
+        A = base.lt([a+b,a-b])
```

Comment:
Can we make this
```suggestion
        A = base.lt([a+b,2*a-b])
```
so that the matrix is not symmetric?